.github/workflows/tests.yml

```diff
@@ -9,19 +9,15 @@ on:
 
 jobs:
 
-  tests:
-
-    runs-on: ubuntu-20.04
-
+  integration-tests:
+    runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
```

Comment:
This is probably now `checkout@v3`:

https://github.com/marketplace/actions/checkout